### PR TITLE
Fix #4531: InputNumber keydown/keyup

### LINKED ISSUE #4531: InputNumber: Disable KeyUp/KeyDown default Support
### Describe the feature you would like to see added

Is there a way to remove default KeyUp/KeyDown event listners from primereact Input Nubmer; Ps: i want event to be propagated and treated by the parent

### PR DESCRIPTION
**Pull request by melloware**
Fix #4531: InputNumber keydown/keyup

Allow user to override key up and keydown to prevent default actions.

### CODE REVIEW

File: components/lib/inputnumber/InputNumber.js [hunk 1]
```diff
@@ -448,27 +457,31 @@ export const InputNumber = React.memo(
...
+
+            const key = event.key;
 
-            if (code !== 13) {
+            if (key !== 'Enter') {
                 // to submit a form
                 event.preventDefault();
             }
 
-            const char = String.fromCharCode(code);
```
Comment (ulasturann) [inline components/lib/inputnumber/InputNumber.js:469]: This definition is missing and is the cause of the error.

File: components/lib/inputnumber/InputNumber.js [hunk 2]
```diff
@@ -448,27 +457,31 @@ export const InputNumber = React.memo(
...
-            }
         };
 
-        const onInputKeyPress = (event) => {
+        const onInputKeyUp = (event) => {
             if (props.disabled || props.readOnly) {
                 return;
             }
 
-            const code = event.which || event.keyCode;
```
Comment (ulasturann) [inline components/lib/inputnumber/InputNumber.js:462]: This definition is missing and is the cause of the error.

**Review by ulasturann — COMMENTED** [review]
Add the missing definitions to prevent the error.

File: components/lib/inputnumber/InputNumber.js (hunk 2 above)
Comment (melloware) [reply components/lib/inputnumber/InputNumber.js:462]: Fixed!

File: components/lib/inputnumber/InputNumber.js (hunk 1 above)
Comment (melloware) [reply components/lib/inputnumber/InputNumber.js:469]: Fixed!

**Pull request merged**